src/glue/connection.ts

```diff
@@ -16,27 +16,88 @@ export enum ConnectionType {
  * Configuration for the Glue Workflow resource.
  */
 export interface ConnectionProps extends ResourceProps {
+  /**
+   * The type of the connection. Currently, these types are supported:
```

Comment:
Rather than commenting the supported types here it'd be better to add the comments to the enum that this references above.